types/newman/index.d.ts

```diff
@@ -173,10 +173,14 @@ export interface NewmanRunExecutionAssertionError {
     test: string;
     message: string;
     stack: string;
-}export interface NewmanRunFailure {
+}
+export interface NewmanRunFailure {
     error: NewmanRunExecutionAssertionError;
     /** The event where the failure occurred */
     at: string;
+    source: any;
+    parent: any;
```

Comment:
:heavy_check_mark:  `parent` is often referred to as any in the [postman collection types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/postman-collection/index.d.ts) so this should be fine